### DISCUSSION TITLE
Link to document version page in copyright footer

### DISCUSSION THREAD

**Discussion by LalteMa** [squidfunk/mkdocs-material#4519] [Q&A]
Hello!

I want to add custom version information at the bottom of the documentation, specific near the copyright information. 
I added these lines in the docs/overrides/partials/copyright.html

`<a href="../../../home/versions/">
      Document Version 2022.10.RC01  
    </a> `

By following the link the user should be directed to a docs/home/versions.md. When serving the documentation the link only works in the case the navigation depth matches the path ../../../home/versions/. 
In the offline build of the documentation the link never works. For example:

**This works!**
Online documentation before the link: `http://localhost:8000/products/`
Online documentation after the link: `http://localhost:8000/home/versions/` 

**This works not!**
Online documentation before the link: `http://localhost:8000/products/product01/features/feature01/`
Online documentation after the link: `http://localhost:8000/products/home/versions/`

**This works not!**
Offline documentation before the link: `C:/Users/User01/Downloads/site-offline/products/index.html`
Offline documentation after the link: `C:/Users/User01home/versions/`

The problem is simply that the link is accessible on every page in every navigation depth, but I set up a static link. 
 How can I link to this page so that I can always access it on the current page, whether it is the online or offline version?

 Thanks in advance!

**Comment by squidfunk** [comment]
I'm sorry, but I don't understand the problem.

**Reply by LalteMa** [reply]
Sorry about that! I have updated the original question!

**Reply by squidfunk** [reply]
Thanks for coming back. I think you would need to set this an absolute URL, not a relative one, or it won't work. It also might not work for offline documentation, since there's no possibility to provide an absolute URL. If you're using `mike`, it might be a good idea to ask the maintainers of the project.
